src/NHSD.BuyingCatalogue.Ordering.Api/Models/CreateOrderItemBaseModel.cs

```diff
@@ -1,35 +1,19 @@
-﻿using System.ComponentModel.DataAnnotations;
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
 using NHSD.BuyingCatalogue.Ordering.Api.Attributes;
+using NHSD.BuyingCatalogue.Ordering.Api.Services.CreateOrderItem;
+using NHSD.BuyingCatalogue.Ordering.Domain;
 
 namespace NHSD.BuyingCatalogue.Ordering.Api.Models
 {
-    public abstract class CreateOrderItemBaseModel
+    [JsonConverter(typeof(CreateOrderItemModelConverter))]
```

Comment:
Should this be applied to CreateOrderItemModel?

Reply:
No, if we apply it to CreateOrderItemModel we get into the same vicious cycle of it trying to deserialize itself over and over again. It should probably be renamed to CreateOrderItemBaseModelConverter though